src/main/java/hudson/security/ProjectMatrixAuthorizationStrategy.java

```diff
@@ -88,20 +87,23 @@ public ACL getACL(ItemGroup g) {
         return getRootACL();
     }
 
+    @Nonnull
+    @Override
+    public ACL getACL(@Nonnull Node node) {
```

Comment:
Like https://github.com/jenkinsci/matrix-auth-plugin/pull/27/commits/e69047a9c6cabfb6d5805d86a5509b4308392891 but adapted for inheritance strategies.